packages/sources/m0/src/index.ts

```diff
@@ -0,0 +1,13 @@
+import { expose, ServerInstance } from '@chainlink/external-adapter-framework'
+import { Adapter } from '@chainlink/external-adapter-framework/adapter'
+import { config } from './config'
+import { nav } from './endpoint'
+
+export const adapter = new Adapter({
+  defaultEndpoint: nav.name,
+  name: 'M0',
+  config,
+  endpoints: [nav],
```

Comment:
You need a rate limiter here otherwise your API is going to get blasted

```
  rateLimiting: {
    tiers: {
      default: {
        rateLimit1m: ???,
        note: '???',
      },
    },
  },
  ```

Reply:
Thanks for review! Just to confirm my understanding: the `rateLimit1m` relates to a single oracle, not all of them right? So if we set to a low number like 1, will multiple oracles be able to request data at the same time?

Reply:
Yes, single oracle

Reply:
added rate limit 1/min

Reply:
Still no rate limiter

Reply:
put it in a wrong place, thanks